app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala

```diff
@@ -285,6 +206,111 @@ class ServiceDependenciesController @Inject()(
       ).merge
     }
 
+  private def getRepositoryInfo(meta: MetaArtefact, latestVersions: Seq[LatestVersion], bobbyRules: BobbyRules) = {
```

Comment:
can we rename to `toRepository` since it transforms data, rather than retrieving anything, and `RepositoryInfo` is not an existing model?